cmd/state-tray/main.go

```diff
@@ -162,7 +162,7 @@ func run() (rerr error) {
 	mReload := mProjects.AddSubMenuItem("Reload", "Reload the local projects listing")
 	localProjectsUpdater := menu.NewLocalProjectsUpdater(mProjects)
 
-	localProjects, err := model.LocalProjects()
+	localProjects, err := model.LocalProjects(context.Background())
```

Comment:
It looks like you use `context.Background()` everywhere.  Is that correct?  I assume that you have experimented with time-out contexts but determined that they are not necessary anymore.

I am fine if we leave this commit (814c6ed55ebe59d5) as-is anyways as it gives us the flexibility to introduce timeouts here.  But I would like to ensure that I understand it correctly that it does not change any behavior right now.

Reply:
This is just preserving the behavior that was there prior to this PR. I just made it more explicit.

Some of these could certainly benefit from a timeout, but they were not the focus of this PR.